### ISSUE TITLE
[Bug / Feature request] MMS with Audio not working

### ISSUE THREAD

**Issue by childnode** [moezbhatti/qksms#1220]
### DESCRIPTION
Using the "Visual Voicemail" (get voicemails as MMS with audio instead of an SMS notification) from Vodafone I experienced that this is not working with QKSMS since it doesn't display the MMS properly.

1. The images are resized to screen width which results in a cluttered, long message with pixeled images
2. (main issue): the attached audio is neither displayed nor is it able to play

### STEPS
1. get an MMS with audio (you might use Android / Google Messages to send one yourself)
2. receive by QKSMS
3. try to hear the message

### EXPECTED
1. [Bug] if MMS contains unsupported (by QKSMS) attachments, a placeholder has to be displayed to show up that there is more, at least allow to save the unsupported MMS attachments to allow user to open them seperately
2. [Feature] Support Audio in MMS (it seems to be a general issue to be, even on new messages I can't add audio, so we might split this issue) 
3. [FUG] Image auto-resize should be set to "shrink to screen width" instead of "fit to screen" 

### OBSERVATIONS
1. neither audio attachment is display nor new MMS with audio are available for writing messages
2. resizing images do not expand small images in attachments

Google Messages:
<img src="https://user-images.githubusercontent.com/192814/41895226-b82cd318-7921-11e8-9d58-3931690d6f1d.png" width="250"/>

QKSMS

<img src="https://user-images.githubusercontent.com/192814/41895227-b86210fa-7921-11e8-991e-69e98970ee58.png" width="250"/>

<img src="https://user-images.githubusercontent.com/192814/41895228-b87fcb4a-7921-11e8-9a71-0c7ce25e4bfc.png" width="250"/>

**Comment by childnode** [comment]
hi @moezbhatti , thanks for your feedback.
just a question in doubt: should I open a second Bug request for the image scaling thing to then retitle this as feature request only?

**Comment by moezbhatti** [comment]
Yeah, I think that the image scaling should be considered a separate issue

**Comment by childnode** [comment]
>> should I open a second Bug request for the image scaling thing
> 
> Yeah, I think that the image scaling should be considered a separate issue

💁‍♂️ spocko was quicker, see #1213

**Comment by NestaYor** [comment]
Just came here from the subreddit and searched this. I never use audio so it wasnt a deal breaker until my father discovered he could send them. hoping he can learn to have google transcribe it for him. 

either way, audio support would be great. i would get "mms message" as a notification but when i opened the sms, it would only show the last message prior to the audio message.
